### PR TITLE
Final method name changes with updates to tutorials/examples

### PR DESCRIPTION
**Pull request by rmw2**
- Renamed `verifyDisclosureResponse` -> `authenticateDisclosureResponse`
- Changed names in tutorials and examples for this and preceding updates
- Updated language about creating app identities to refer to new app configuration wizard

### CODE REVIEW

File: docs/guides/tutorial.md [hunk 1]
```diff
@@ -34,17 +41,17 @@ var credentials = new uport.Credentials({
...
+When we hit the default route using `app.get('/')` we will call `credentials.createVerification()` in order to sign the credential. For the fields of the credential, the `sub` field is the subject. Set this to the uPort Id of the user that is supposed to receive the credential. For testing purposes this would be the uPort identity shown on the mobile app of the reader. The `exp` field is the expiry of the token, in Unix time (seconds precision). As `claim` field, put your own custom object. We show an example below. The format of the claim needs to be `{'Title': {'key':'value', 'another key': 'another value', ...}}` or simply `{'Title' : 'Value'}`. We do not support more nested claims at this time.
```
Comment (pelle) [inline docs/guides/tutorial.md]: We actually do support nested claims now. See the format used by the App Manager or on demo.uport.space

File: docs/guides/tutorial.md [hunk 2]
```diff
@@ -13,19 +13,26 @@ Here we will demonstrate how to create and sign a custom credential on a server
...
+This tutorial uses sample application identities (i.e. private keys) to issue and verify credentials on a server.  For your own applications, you can create a new private key with this library, using the code snippet below:
+```
+$ node
+> const { Credentials } = require('uport')
+> Credentials.createIdentity()
+{
+  did: 'did:ethr:0x123...',
+  privateKey: '3402abe3d...'
+}
+```
```
Comment (localredhead) [inline docs/guides/tutorial.md]: :+1:

File: docs/guides/tutorial.md [hunk 3]
```diff
@@ -13,19 +13,26 @@ Here we will demonstrate how to create and sign a custom credential on a server
...
 
-*Please note that in practice the signing key for the identity should be protected information*
+*Please note that in practice the signing key for the identity should be kept secret!*
```
Comment (localredhead) [inline docs/guides/tutorial.md]: :100:

File: docs/guides/tutorial.md [hunk 4]
```diff
@@ -13,19 +13,26 @@ Here we will demonstrate how to create and sign a custom credential on a server
...
 
 ## Creator service
 
 In the file `createcredential.js` we have a simple node `express` server. In the setup phase we use the private key we got from the App Manager to create a `SimpleSigner` object.
 
-We then create a `Credentials` object using the privatreKey and the uPort identifier of our app that we got from the App Manager (or the default identity):
+We then create a `Credentials` object using the privateKey and the uPort identifier of our app that we got from the App Manager (or the default identity):
```
Comment (localredhead) [inline docs/guides/tutorial.md]: ;)

**Comment by rmw2** [comment]
@pelle Regarding `verifyDisclosure` vs. `verifyDisclosureResponse`, the thought was that it doesn't necessarily need to be a response to a disclosure request -- as with the old name of `verifyProfile`, we wanted to indicate that it could come from another source, such as a public profile or be shared by a third party. That was the thought for the word `response` in what is now `authenticateDisclosureResponse`.  At one point we floated the idea of `verifyArbitraryDisclosure`  to better indicate that it didn't have to be a response, but I think `authenticateDisclosureResponse` and `verifyDisclosureResponse` might be confusing.  I think all things considered I'm most happy with how it is now (`authenticateDisclosureResponse` and `verifyDisclosure`), but happy to consider other options, or to add `response` to `verifyDisclosure` if you feel strongly.

For the nested claims, I'll fix that now -- do we have any specific limitations in the spec as of now?

**Comment by rmw2** [comment]
Only worry is what if it isn't a "profile" per se, and we don't use that word anywhere else so it feels a tiny bit like adding more jargon that might be hard to parse.

**Comment by pelle** [comment]
@rmw2 you convinced me. verifyDisclosure() it is 👍
